docs/advanced_usage.md

```diff
@@ -113,6 +113,62 @@ Yubikey support requires
 (which are bundled with the PIV tools)</a> to be available in standard
 library locations.
 
+### Use the native store
+
+We currently support native storage on OS X, Linux and Windows. The advantage of using the native store to manage keydata is 3-fold. It is more easily visible, it more secure and it is more convenient- since the native store is secured, there is no need for passphrases on keys. You can enable use of the native store by following the below instructions. The programs in this repository are written with the Go programming language. These instructions assume that you have set it up your machine. 
```

Comment:
some small nits on this line:
 - small typo: `it is more secure`
 - I think you should elaborate on "since the native store is secured" - you should mention how it already has OS-level access control (not sure how to best phrase this)
 - also, have we tested wincred?  If not, we should not release it